CodeMaid/Logic/Cleaning/FileHeaderLogic.cs

```diff
@@ -14,6 +15,8 @@ internal class FileHeaderLogic
 
         private readonly CodeMaidPackage _package;
 
+        private const int HeaderMaxNbLines = 30;
```

Comment:
Any rationale behind 30 lines or just needed an upper boundary?  Any thoughts on how to make this limit more transparent to users?

Reply:
No rationale, just looked fine to me. I just doubled it so it should be no problem, 60 lines is huge for a header.

I could add that as a tooltip somewhere but I really don't think users would benefit from this information. We could adjust it in case of bug report